order/batch.go

```diff
@@ -298,9 +298,12 @@ func AbandonCanceledChannels(matchedOrders map[Nonce][]*MatchedOrder,
 				},
 			)
 			if err != nil {
-				log.Warnf("Unable to abandon channel "+
-					"(channel_point=%v:%d) for order=%v",
-					txHash, idx, ourOrderNonce)
+				log.Infof("Cleaning up incomplete/replaced " +
```

Comment:
:+1:

Reply:
Consider debug instead?

Reply:
Should we check the error message here somehow? To know that we got an expected error, not a critical one

Reply:
Wasn't sure if it would ever be helpful in case do run into problem and this fails for an actual reason. But I guess we'll ask people to run with `debuglevel=debug` in case of a problem anyway, so I changed this to `Debugf`.

Reply:
we could string match (or something better?) on the expected error before logging or exiting in case of a critical one.